mifosng-android/src/main/java/com/mifos/mifosxdroid/injection/module/ApplicationModule.kt

```diff
@@ -18,27 +18,6 @@ import javax.inject.Singleton
 @InstallIn(SingletonComponent::class)
 class ApplicationModule {
 
-//    @Provides
```

Comment:
why did you commented this code?

Reply:
While implementing multi-module, I commented this code but forgot to remove it.